python_modules/dagster/dagster/_core/run_coordinator/queued_run_coordinator.py

```diff
@@ -204,6 +237,35 @@ def config_type(cls) -> UserConfigSchema:
                     " launcher is being used."
                 ),
             ),
+            "block_op_concurrency_limited_runs": Field(
+                config=bool,
+                is_required=False,
+                description=(
+                    "Determines whether or not a run will be dequeued if it consists of ops that "
+                    "will all be blocked waiting for global op concurrency slot to be free."
+                ),
+            ),
+            "op_concurrency_slot_offset": Field(
+                config=IntSource,
+                is_required=False,
+                default_value=0,
+                description=(
+                    "Determines the number of pending runs waiting for concurrency slots allowed "
+                    "when deciding to dequeue a run. Only used block_op_concurrency_limited_runs "
+                    "is True."
+                ),
+            ),
+            "op_concurrency_runs_started_buffer_seconds": Field(
+                config=IntSource,
+                is_required=False,
+                default_value=0,
```

Comment:
should the default value here be higher than 0? Generally speaking this should be a pretty constant and short time though, so i'm wondering if this should be an env var or something that can be tweaked if you really know what you're doing, like fixed at 5-10 seconds or something?

Reply:
yeah... my guess is that it's probably okay to have this be near 0.   You might be off a little and have a spinning run here or there, but it wouldn't be too bad.  

I think maybe I should get rid of this feature or bury it as an env var, but that doesn't feel 100% correct either, since the value is read in the host cloud daemon, and the latency will be determined by the per-org run worker environment (e.g. how many resources does the run worker have, how long will it take the executor to process its first level of steps, etc).

Reply:
I would support "bury as an env var defaulted to 5 seconds"